site/themes/s2b_hugo_theme/layouts/partials/cal/pp-feed.html

```diff
@@ -1,5 +1,5 @@
 <div id="pp-calendar-feed" class="pp-banner" style="text-align: center; padding-top: 1rem; margin-top: 1rem;">
-  <p>Want to see rides in your own calendar? Subscribe to the calendar feed in the calendar app on your phone or computer!</p>
-  <p><button type="button" id="feed-sub" class="btn">Copy calendar feed URL</button></p>
-  <p><input type="text" id="feed-url" readonly value="https://www.shift2bikes.org/cal/pedalpalooza-calendar.php"></p>
+  <p>Want to see rides using your computer or phone's calendar app? 
```

Comment:
`</p>`

Reply:
is that needed, or just a sort of best practice thing?
( i thought generally `/p`'s weren't needed, and that the `/div` right after should close it up. )
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p